front/App.tsx

```diff
@@ -47,7 +47,7 @@ const App: FC = () => {
       });
   }, []);
 
-  if (process.env.CLEAR_STORAGE)
+  if (process.env.CLEAR_STORAGE && process.env.CLEAR_STORAGE === "true")
```

Comment:
Pourquoi pas tout simplement ?
```suggestion
  if (process.env.CLEAR_STORAGE === "true")
```